RBAC/router.py

```
@@ -0,0 +1,348 @@
+    - List[OrganizationRead]: A list of organizations assigned to the current user.
+
+    Raises:
+    - HTTPException: If there is an error during the database query execution, a 500 Internal Server Error is raised.
+    """
+    try:
+        result = await db.execute(
+            select(Organization).join(OrganizationUser).where(OrganizationUser.user_id == current_user.id)
+        )
+        organizations = result.scalars().all()
+        return [OrganizationRead.from_orm(org) for org in organizations]
+    except Exception as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)